version

```diff
@@ -1 +1 @@
-2.1.0
+2.2.0
```

Comment:
The version upgrades are at the moment handled as part of different PRs. This is because the release process for new versions of the OpenAPI Terraform plugin creates the release chagelog automatically based on the PRs opened from the last release to the new one being created.

Reply:
Will the release process trigger on non-master branches? And will it fire if your 2.2.0 release has already occurred?

The only reason the version increment is present is because I pulled in your master branch and the PR is targeting your repo's `bradthebuilder-master` branch that doesn't yet have master's latest updates. I was just trying to simplify an eventual merge of your `bradthebuilder-master` into your `master` branch (at which point you wouldn't see a version increase, because the versions would match).  I can either:

1. Make a new PR directly against your `master` branch so the version increment doesn't show up as a change
2. Make a new branch in my repo _without_ your `master's` latest version increase and then create a new PR still targeting your `bradthebuilder-master` branch so the version increment won't be present

Whatever is easiest for you!

Reply:
+1 on the first option, I think it'd be cleaner creating another PR right from master and we will only see the new things added.